docs/docusaurus/docs/reference/expectations/result_format.md

```diff
@@ -3,67 +3,74 @@ title: Result format
-```
+To apply `result_format` to an Expectation, you pass it into the Expectation. 
+
+1. Run `$ great_expectations suite new` to obtain a Validator object instance.
+
+2. Run the following code to pass `result_format` into the Expectation:
+
+    ```python name="tests/integration/docusaurus/reference/core_concepts/result_format/result_format_complete_example_set"
+    ```
 
 ### Checkpoint-level configuration
-To apply `result_format` to every Expectation in a Suite, define it in your Checkpoint configuration under the `runtime_configuration` key.
+
+Run the following code to apply `result_format` to every Expectation in a Suite:
 
 ```python name="tests/integration/docusaurus/reference/core_concepts/result_format/result_format_checkpoint_example"
 ```
+Your Checkpoint configuration is defined below the `runtime_configuration` key. 
+
+The results are stored in the Validation Result after running the Checkpoint.
 
-The results will then be stored in the Validation Result after running the Checkpoint.
 :::note
 The `unexpected_index_list`, as represented by primary key (PK) columns, is 
-rendered in DataDocs when `COMPLETE` is selected. 
+rendered in Data Docs when `COMPLETE` is selected. 
 
-The `unexpected_index_query`, which for `SQL` and `Spark` is a query that allows you to retrieve the full set of 
-unexpected values from the dataset, is also rendered by default when `COMPLETE` is selected. For `Pandas`, this parameter 
-returns the full set of unexpected indices, which can also be used to retrieve the full set of unexpected values. This is returned
-whether or not the `unexpected_index_column_names` are defined. 
+The `unexpected_index_query`, which for `SQL` and `Spark` is a query that allows you to retrieve the full set of unexpected values from the dataset, is also rendered by default when `COMPLETE` is selected. For `Pandas`, this parameter returns the full set of unexpected indices, which can also be used to retrieve the full set of unexpected values. This is returned
+whether the `unexpected_index_column_names` are defined. 
 
-To suppress this output, the `return_unexpected_index_query` parameter can be set to `False`. 
+To suppress this output, set the `return_unexpected_index_query` parameter to `False`. 
 
 Regardless of how Result Format is configured, `unexpected_list` is never rendered in Data Docs.
 :::
 
-## Result format values and fields
-
+## Expectations result format values and fields
 
-
-### All Expectations
 | Fields within `result`                | BOOLEAN_ONLY                       |BASIC           |SUMMARY         |COMPLETE        |
 ----------------------------------------|------------------------------------|----------------|----------------|-----------------
 |    details (dictionary)               | Defined on a per-Expectation basis |
-### Column map Expectations (e.g. `ColumnMapExpectation`, `ColumnPairMapExpectation`, `MulticolumnMapExpectation`)
+
```

Comment:
For some reason I can't edit this section, but it should be removed.

Reply:
@austiezr I'm not clear what you want removed.  Can you clarify?